docs/api-reference/core/controller.md

```diff
@@ -14,7 +14,7 @@ The base Controller class supports the following options:
 * `dragRotate` (`Boolean`) - enable rotating with pointer drag. Default `true`
 * `doubleClickZoom` (`Boolean`) - enable zooming with double click. Default `true`
 * `touchZoom` (`Boolean`) - enable zooming with multi-touch. Default `true`
-* `touchRotate` (`Boolean`) - enable rotating with multi-touch. Default `false`
+* `touchRotate` (`Boolean`) - enable rotating with multi-touch. Use two-finger rotating gesture for horizontal and three-finger swiping gesture for vertical rotation. Default `false`
```

Comment:
Q: Useful to enable this separately `verticalRotate` etc?

Reply:
The `dragRotate` switch on desktop controls both pitch and bearing.